MIGRATION.md

```diff
@@ -32,27 +32,81 @@ var subscription = myObservable.subscribe(doSomething);
 subscription.unsubscribe();
 ```
 
-
 ## Subscription: All Subscriptions Are "Composite"
 
 In RxJS 4, there was the idea of a `CompositeSubscription`. Now all Subscriptions are "composite".
 Subscription objects have an `add` and `remove` method on them useful for adding and removing subscriptions
 enabling "composite" subscription behavior.
 
-
-
-## Operators Renamed
+## Operators Renamed or Removed
 
 |RxJS 4|RxJS 5|
 |---|---|
-|`flatMap`|`mergeMap` or `flatMap`(alias)|
-|`flatMapLatest`|`switchMap`|
-|`bufferWithTime`|`bufferTime`|
 |`bufferWithCount`|`bufferCount`|
+|`bufferWithTime`|`bufferTime`|
+|`flatMap` or `selectMany`|`mergeMap` or `flatMap`(alias)|
+|`flatMapFirst`|`exhaustMap`|
+|`flatMapLatest`|`switchMap`|
+|`flatMapWithMaxConcurrent`|`mergeMap` or `flatMap`(alias)|
+|`publishValue`|`publishBehavior`|
+|`replay`|`publishReplay`|
+|`select`|`map`|
+|`selectConcat`|`concatMap`|
+|`switchFirst`|`exhaust`|
+|`tap`|`do`|
 |`windowWithTime`|`windowTime`|
 |`windowWithCount`|`windowCount`|
-|`switchFirst`|`exhaustFirst`|
-|`switchFirstMap`|`exhaustFirstMap`|
+|`where`|`filter`|
+|`amb`|`-`|
+|`and`|`-`|
```

Comment:
what about use term like (WIP) or similar for now except operators completely decided to be dropped? might possible some people think lot of operators are dropped by just reading this document for first time.